src/Util/PostgresHelper.php

```diff
@@ -0,0 +1,33 @@
+<?php
+declare(strict_types=1);
+
+namespace Prooph\EventStore\Pdo\Util;
+
+trait PostgresHelper {
+    /**
+     * @param string $tableName
+     * @return string
+     */
+    private function quoteIdent(string $tableName): string
+    {
+        return array_reduce(explode('.', $tableName), function ($result, $part) {
```

Comment:
that's really hard to read, can we simplify that?

Reply:
ready